### PR TITLE
Added SBOM creation as github action

### PR DESCRIPTION
**Pull request by schaefi**
Make use of the cargo-bom tool to create a bill of materials such that it becomes possible to check CVE information against the build of oci-pilot

### CODE REVIEW

File: .github/workflows/ci-sbom.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install
+        run: |
+          sudo apt update && sudo apt install cargo
+          cargo install cargo-bom
+      - name: Run SPDX Creation for oci-ctl
+        run: |
+          cd oci-ctl
+          cargo bom
```
Comment (isbm) [inline .github/workflows/ci-sbom.yml:23]: Shouldn't it be like `cargo bom > BOM.txt` instead? So it produces something [like this](https://github.com/sensorfu/cargo-bom/blob/HEAD/BOM.txt)?

**Review by isbm — CHANGES_REQUESTED** [review]
Just a question.

File: .github/workflows/ci-sbom.yml (hunk 1 above)
Comment (schaefi) [reply .github/workflows/ci-sbom.yml:23]: well you said we should have something for later. There is currently no consumer of this information and also nobody can tell in what shape or form it should be delivered. Therefore please don't let us populate this data in any other place than the github action artifacts. That's enough for the moment. Thanks

Comment (isbm) [reply .github/workflows/ci-sbom.yml:23]: That's a very good question what to do _next_ with it. 😉 Let's follow up then.

**Review by isbm — APPROVED** [review]

File: .github/workflows/ci-sbom.yml (hunk 1 above)
Comment (schaefi) [reply .github/workflows/ci-sbom.yml:23]: yep

**Pull request merged**